org.openscopeproject.TrguiNG.yml

```diff
@@ -0,0 +1,121 @@
+id: org.openscopeproject.TrguiNG
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: trgui-ng
+rename-icon: trgui-ng
+copy-icon: true
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+  - org.freedesktop.Sdk.Extension.rust-stable
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --filesystem=home
```

Comment:
home is a broad filesystem permission

Reply:
The app needs to be able to read and write torrent files. Most commonly from the downloads location but user should be able to save their created torrent file anywhere.

Reply:
> The app needs to be able to read and write torrent files. Most commonly from the downloads location but user should be able to save their created torrent file anywhere.

Does it support filechoosers? Then you can open torrents from anywhere but save it to Downloads only.

Reply:
It does use filechoosers but if program is invoked from file manager by clicking torrent files it will not be invoked. Does path remapping still happen in such cases?

Reply:
Tested this some more, opening files from file managers also works fine.

But problem is when trying to choose save location for saved torrent, if it's outside of permitted filesystem it turns into path that transmission will not understand (which is not running inside of flatpak). So this adds significant hurdle to trying to use one of the basic features of the app.

Reply:
Can you add a default save location to `~/Downloads` and check if the location is writeable before saving? 

If someone needs extra locations they can add the permission. "home" isn't a catch-all, someone may want to save to an external drive. You can't cover every location.